src/main/scala/DemoFortis.scala

```diff
@@ -134,7 +137,7 @@ object DemoFortis {
           .map(post => {
             val source = post.post.getPermalinkUrl.toString
             val language = languageDetection.detectLanguage(post.post.getMessage)
-            val analysis = Analysis(language = language)
+            val analysis = Analysis(language = language, keywords = keywordExtractor.extractKeywords(post.post.getMessage))
```

Comment:
We may want to do the keyword extractions as a separate map step instead of merged with another transform. This has a few advantages:
1. Conceptually simpler: every map just does one thing
2. We can more easily switch different processing steps on or off by just removing a map call
3. The map operations are independent so can be done in parallel